data-raw/process_data_sets.R

```diff
@@ -272,3 +272,14 @@ pennies_resamples <-
   select(replicate, everything()) %>%
   unnest(cols = c(data))
 usethis::use_data(pennies_resamples, overwrite = TRUE)
+
+#-------------------------------------------------------------------------------
+# Babies: Documented in R/babies.R
+#-------------------------------------------------------------------------------
+# Population of 1236 babies from
+# https://wjhopper.github.io/SDS-201/data/babies.csv
+babies <- "https://wjhopper.github.io/SDS-201/data/babies.csv" %>%
+  read_csv() %>%
+  mutate(birthday = date) %>% # Creating new date variable
```

Comment:
You don't need line 283. Line 284 will create `birthday` with the `date` in the right format. Also, do you want the original `date` variable in the dataset? If not, please remove it.